### PR TITLE
Update redis client gem

### PR DESCRIPTION
**Pull request by valikos**
Update redis client gem

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 language: ruby
 rvm:
-  - 2.5.0
+  - 2.5.8
 script: bundle exec rspec
+gemfile:
+  - Gemfile-rails-4.2.x
```
Comment (Liinal) [inline .travis.yml:6]: Do we need 4.2?

**Review by Liinal — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (andriymosin) [reply .travis.yml:6]: Nope

**Review by andriimosin — APPROVED** [review]
🤞

File: Gemfile-rails-4.2.x [hunk 2]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gemspec
+
+gem 'activerecord', '< 5.0'
+gem 'sqlite3',      '~> 1.3.0'
```
Comment (Groovej) [inline Gemfile-rails-4.2.x:6]: I think we can skip 4.2 version.

**Review by Groovej — COMMENTED** [review]
Looks good, just remove rails 4.2 support. 🚀

**Review by Groovej — APPROVED** [review]

**Pull request merged**